src/hooks/actions.tsx

```diff
@@ -319,29 +319,6 @@ export function useProposeAction({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [proposePriceTransaction]);
 
-  if (
-    query?.identifier === "MULTIPLE_VALUES" &&
-    !isMultipleValuesInputValid(
-      proposePriceInput,
-      query.proposeOptions?.length ?? 0,
-    )
-  ) {
-    return {
-      title: propose,
-      disabled: true,
-      disabledReason: 'Either enter all fields, or mark "Unresolvable"',
-    };
-  }
-
-  // TODO validate input
-  if (!isInputValid(proposePriceInput ?? "")) {
-    return {
-      title: propose,
-      disabled: true,
-      disabledReason: "Please enter a value to propose.",
-    };
-  }
-
```

Comment:
This was causing a bug in the PrimaryAction button. 
We only want to do this validation if on propose page